components/molecule/modal/src/index.scss

```diff
@@ -182,31 +193,4 @@ body.is-MoleculeModal-open {
       width: $w-molecule-modal-icon;
     }
   }
-
-  &-content {
-    -webkit-overflow-scrolling: touch;
-    flex: 1 1 auto;
-    overflow-y: auto;
-    margin: $mt-molecule-modal-content $mr-molecule-modal-content 0
```

Comment:
Changed this to use padding instead because of the scroll. I think that it won't cause any breaking change. Any suggestion @andresin87 ?

Reply:
checked